### PR TITLE
Dockerized frontend

### LINKED ISSUE #120: Dockerize Frontend + Deploy Web

### PR DESCRIPTION
**Pull request by Patricijia**

### CODE REVIEW

**Review by VictorVader — DISMISSED** [review]

File: frontend/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: "3.8"
+
+services:
+  app:
+    container_name: docker-react-container
+    image: docker-react-i
+    build:
+        context: .
+    ports:
+        - 3000:30
```
Comment (NeaGogu) [inline frontend/docker-compose.yml]: delete this file

File: frontend/.dockerignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+*.Dockerfile
+*.md
+*/node_modules
```
Comment (NeaGogu) [inline frontend/.dockerignore]: move this to project root

**Review by NeaGogu — CHANGES_REQUESTED** [review]

File: frontend/docker-compose.yml (hunk 1 above)
Comment (Patricijia) [reply frontend/docker-compose.yml]: @NeaGogu wait the entire file?

Comment (NeaGogu) [reply frontend/docker-compose.yml]: do you use it?

Comment (NeaGogu) [reply frontend/docker-compose.yml]: if no, then yes :laughing:

**Review by NeaGogu — APPROVED** [review]

**Review by VictorVader — APPROVED** [review]

**Pull request merged**